### PR TITLE
[SYCL][HIP] Mark select group algorithm tests as xfail for hip/amd

### PR DESCRIPTION
**Pull request by cperkinsintel**
the GroupAlgorithm tests were recently expanded to ensure that all the spir-v 1.3 binary ops were being exercised. Unfortunately, this seems to have introduced a failure on hip/amd. Marking these tests as xfail temporarily to unblock other PRs

Signed-off-by: Chris Perkins <chris.perkins@intel.com>

### CODE REVIEW

**Review by steffenlarsen — DISMISSED** [review]

File: SYCL/GroupAlgorithm/SYCL2020/exclusive_scan.cpp [hunk 1]
```diff
@@ -3,6 +3,8 @@
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
 
+// XFAIL: hip
```
Comment (steffenlarsen) [inline SYCL/GroupAlgorithm/SYCL2020/exclusive_scan.cpp:8]: Should we mention the reason why they fail here? It is due to a selection of undefined symbols for `__spirv_*` functions, right?

Comment (cperkinsintel) [reply SYCL/GroupAlgorithm/SYCL2020/exclusive_scan.cpp:8]: done

**Comment by pvchupin** [comment]
@cperkinsintel, can you open an https://github.com/intel/llvm/issues for these to reenable back?

**Review by steffenlarsen — APPROVED** [review]

**Review by v-klochkov — APPROVED** [review]

**Comment by cperkinsintel** [comment]
issue opened: https://github.com/intel/llvm/issues/6185

**Pull request merged**